bs-react-native-next/src/apis/TimePickerAndroid.re

```diff
@@ -0,0 +1,29 @@
+type options;
+
+[@bs.obj]
+external options:
+  (
+    ~hour: int=?,
+    ~minute: int=?,
+    ~is24Hour: bool=?,
+    ~mode: [@bs.string] [ | `clock | `spinner | `default]=?,
+    unit
+  ) =>
+  options =
+  "";
+
+type response = {
+  .
+  action: string,
+  hour: int,
+  minute: int,
+};
+
+[@bs.module "react-native"] [@bs.scope "TimePickerAndroid"]
+external _open: options => Js.Promise.t(response) = "open";
```

Comment:
shouldn't we do `open_` instead ? I remember that reason-react change some thing `_likethat` in favor of things `likethat_` not sure why.

Reply:
fixed

Reply:
Further to my comment in #403, perhaps this should really be `_open`?

Reply:
Right, for object labels (`f##_open` in your link) and for bs.obj labelled arguments we need to have the underscore prefix, it won't work any other way.

For externals we are free to decide, and as the leading `_` usually means "unused", I am also in favor of `open_` in this case.

Reply:
Would be nice to have bs.obj to work with suffix instead of prefix...
I would prefer `open_` too... :'(